playbook/app/pb_kits/playbook/pb_date_picker/_date_picker.jsx

```diff
@@ -40,7 +40,7 @@ type DatePickerProps = {
   onChange: (String) => void,
   pickerId?: String,
   placeholder?: String,
-  plugins?: Boolean,
+  plugins?: "ms" | "ws",
```

Comment:
Apparently this `plugins` prop was not implemented in a very sensible way from my perspective. The way it currently stands, this boolean simply [disables one plugin that is being configured inside the helper itself](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_date_picker/date_picker_helper.js#L56). I am having trouble understanding the direction of the original code you are replacing as it seem a lot more reasonable, from a Playbook kit perspective, to instead have props that do a better job of explaining what their purpose is compared to just `plugins`. These props would simply map to each individual feature. Let's your change here as an example:

```tsx
// instead of this plurally named enumerable
plugins?: "ms" | "ws"
```
```tsx
// add individual props, one for each feature
monthSelect?: boolean,
weekSelect?: boolean,
```

This would still achieve the same effect, but be a lot less vague than the original implementation. If you chose to do this as described above, you still have to fool-proof it a little bit incase someone enables two or more features that do not work together, so perhaps you could then do some consolidation:

```tsx
// add a selection type feature
selectionType?: "month" | "week"
```

In summary, this would be the best option which can both better explain what is going in to the kit and what will be produced without introducing any crossing feature paths like month vs. day selection.